test-tests/shared/src/test/scala/zio/test/AssertionSpec.scala

```diff
@@ -569,7 +569,21 @@ object AssertionSpec extends ZIOBaseSpec {
       val assertionM = AssertionM.assertionDirect[Unit]("sameName")()(_ => ???)
       assert(assertion.equals(assertionM))(isFalse ?? "assertion != assertionM") &&
       assert(assertionM.equals(assertion))(isFalse ?? "assertionM != assertion")
-    }
+    },
+    test("isThrowableCause must succeed when supplied value has matching cause") {
```

Comment:
"hasThrowableCause" here?

Reply:
:facepalm: Yep, hasThrowableCause, fixed in the commit below.

Reply:
Awesome!